### DISCUSSION TITLE
Saving sort by settings and moving extruder after canceling a print

### DISCUSSION THREAD

**Discussion by bchau21** [UnchartedBull/OctoDash#1349] [Q&A] [answered]
Is there a way to save the sort by settings? I prefer to view the files by date and descending. It always reverts back to the default settings (name/ascending). 

Also, is there a way to make the extruder move up after I've canceled a print via OctoDash? Currently, the extruder stays where it's at when I cancel the print. If i'm not using OctoPrint and I cancel a print via the screen on my MK3S, the extruder moves up and back to the left.

**Comment by UnchartedBull** [comment, marked as answer]
It's available in the settings.

OctoPrint cancel GCode are your best friends here. This is the one I'm using, feel free to copy and adjust:

```
M104 S0 ; turn off temperature
M140 S0 ; turn off heatbed
M107 ; turn off fan
G91; relative positioning
G1 Z10 F600; Move print head up
G1 E-10; retract
G90; absolute positioning
G1 X0 Y200 F3000 ; present print
M84 X Y E ; disable motors
```

**Reply by bchau21** [reply]
Ah, missed the settings somehow. 

Thanks for the help!
